0.10/s2i/bin/run

```diff
@@ -8,7 +8,7 @@ run_node() {
   if [ "$DEV_MODE" == true ]; then
     exec nodemon --debug="$DEBUG_PORT"
   else
-    exec npm start -d
+    exec npm run $NPM_RUN
```

Comment:
should this just be "npm $NPM_RUN" ?  and the default value of NPM_RUN needs to be "start -d"?


also if you're adding a new env variable, please doc it in the README (and ideally in openshift-docs as well, but i can take care of that part).